dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/EPersonRestRepository.java

```diff
@@ -44,6 +45,9 @@
     @Autowired
     AuthorizeService authorizeService;
 
+    @Autowired
+    DSpaceObjectMetadataPatchUtils metadataPatchUtils;
```

Comment:
It's unclear why this is now `@Autowired` and imported into this class. I don't see any new code that uses it?  Are these changes to `EPersonRestRepository` necessary?

Reply:
You are correct @tdonohue, this should not be present in this part of the code (Seems to be a remnant of an older commit that has been overlooked)
I'll get these feedback-points looked at. Thanks for the review